handlers.go

```diff
@@ -237,12 +237,12 @@ func (s *Server) ServeAPI() error {
 	netw.GET("/peers", s.handleNetPeers)
 	netw.GET("/addrs", s.handleNetAddrs)
 
-	miners := public.Group("/miners")
-	miners.GET("", s.handleAdminGetMiners)
-	miners.GET("/failures/:miner", s.handleGetMinerFailures)
-	miners.GET("/deals/:miner", s.handleGetMinerDeals)
-	miners.GET("/stats/:miner", s.handleGetMinerStats)
-	miners.GET("/storage/query/:miner", s.handleQueryAsk)
+	storageProviders := e.Group("/storage-providers")
+	storageProviders.GET("", s.handleAdminGetStorageProviders)
+	storageProviders.GET("/:sp/deals", s.handleSpGetDeals)
```

Comment:
nit: i prefer being verbose so `handleSpGetDeals` -> `handleStorageProviderGetDeals`, if no one disagrees. same with the other handlers here